Assets/LeapMotionModules/Interaction/InteractionBehaviourBase.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Leap;
+using Leap.Unity.Attributes;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.UI.Interaction {
+
+  public abstract class InteractionBehaviourBase : MonoBehaviour {
+
+    public InteractionManager interactionManager;
+
+    protected Rigidbody _body;
+    public new Rigidbody rigidbody { get { return _body; } }
+
+    [Header("Interaction Settings")]
+    public bool allowHover = true;
```

Comment:
Small thing, but it seems strange that it is called allowHover here, but enableHover in the manager.